HealthTracker/src/data/repositories/geminiRepository.ts

```diff
@@ -0,0 +1,53 @@
+import geminiApiClient from "../../api/geminiApiClient";
+import { EXTRACT_BLOOD_PRESSURE_PROMPT } from "../../constants/geminiPrompts";
+import { GeminiResponse } from "../../api/types";
+import { BloodPressureReading } from "../../domain/models/bloodPressureReading";
+
+export const geminiRepository = {
+  getReadingsFromImage: async (
+    imageBase64: string
+  ): Promise<BloodPressureReading> => {
+    const requestBody = {
```

Comment:
move this in to the types.ts in order to have a better distribution of the request